microk8s-resources/actions/enable.juju.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eu
+
+source $SNAP/actions/common/utils.sh
+
+
+function get_juju_client () {
+  # check if juju cli is already in the system.  Download if it doesn't exist.
+  if [ ! -f "${SNAP_DATA}/bin/juju" ]; then
+    sudo mkdir -p "$SNAP_DATA/bin"
+    sudo mkdir -p "$SNAP_DATA/tmp"
+    sudo "${SNAP}/usr/bin/curl" -L https://launchpad.net/juju/2.6/2.6.4/+download/juju-2.6.4-centos7.tar.gz -o "$SNAP_DATA/tmp/juju.tar.gz"
```

Comment:
Hopefully the juju people will be able to point us to a smaller tarball. Thanks @wallyworld :)

Reply:
Yes, we will look to do a k8s only CLI client, hopefully will be approx 12MB